package.json

```diff
@@ -27,37 +27,37 @@
     },
     "homepage": "https://github.com/ChainSafe/bls-hd-key#readme",
     "devDependencies": {
-        "@babel/cli": "^7.2.3",
-        "@babel/core": "^7.3.3",
-        "@babel/preset-env": "^7.3.1",
-        "@babel/preset-typescript": "^7.3.3",
-        "@babel/register": "^7.0.0",
-        "@types/bn.js": "^4.11.5",
-        "@types/chai": "^4.1.7",
+        "@babel/cli": "^7.8.3",
+        "@babel/core": "^7.8.3",
+        "@babel/preset-env": "^7.8.3",
+        "@babel/preset-typescript": "^7.8.3",
+        "@babel/register": "^7.8.3",
+        "@types/bn.js": "^4.11.6",
+        "@types/chai": "^4.2.10",
         "@types/mocha": "^5.2.5",
-        "@types/node": "^12.0.10",
-        "@typescript-eslint/eslint-plugin": "^1.3.0",
-        "@typescript-eslint/parser": "^1.3.0",
+        "@types/node": "^13.7.7",
+        "@typescript-eslint/eslint-plugin": "^2.22.0",
+        "@typescript-eslint/parser": "^2.22.0",
         "chai": "^4.2.0",
-        "eslint": "5.16.0",
-        "eslint-plugin-import": "^2.18.2",
+        "eslint": "6.8.0",
+        "eslint-plugin-import": "^2.20.1",
         "karma": "^4.4.1",
         "karma-chai": "^0.1.0",
         "karma-chrome-launcher": "^3.1.0",
         "karma-cli": "^2.0.0",
         "karma-mocha": "^1.3.0",
         "karma-spec-reporter": "^0.0.32",
         "karma-webpack": "^4.0.2",
-        "mocha": "^6.1.4",
+        "mocha": "^6.2.2",
         "ts-loader": "^6.2.1",
-        "ts-node": "^8.3.0",
-        "typescript": "^3.5.1",
-        "webpack": "^4.41.2"
+        "ts-node": "^8.7.0",
+        "typescript": "^3.8.3",
+        "webpack": "^4.42.0"
     },
     "dependencies": {
         "assert": "^2.0.0",
```

Comment:
Didn't we get issue that assert polyfill sucks in browser?

Reply:
I'm not sure, happy to remove it in another PR tho!

Reply:
https://github.com/ChainSafe/gossipsub-js/pull/65